models/games.models.js

```diff
@@ -43,10 +43,47 @@ exports.fetchReviews = () => {
 exports.fetchReviewsById = (review_id) => {
 
     return db.query(`
-        SELECT * FROM reviews
+        SELECT * FROM comments
         WHERE review_id = $1;
-    `, [review_id]).then((reviews) => {
+    `, [review_id]).then((comments) => {
 
+        if (comments.rows.length === 0){
+            return db.query(`
+            SELECT 
+                reviews.review_id as review_id,
+                reviews.title as title,
+                reviews.designer as designer,
+                reviews.owner as owner,
+                reviews.review_img_url AS review_img_url,
+                reviews.review_body AS review_body,
+                reviews.category AS category,
+                reviews.created_at AS created_at,
+                reviews.votes as votes,
+                0 AS comment_count
+            FROM reviews
+            WHERE review_id = $1;
+            `, [review_id])
+        } else {
+            return db.query(`
+            SELECT 
+                reviews.review_id as review_id,
+                reviews.title as title,
+                reviews.designer as designer,
+                reviews.owner as owner,
+                reviews.review_img_url AS review_img_url,
+                reviews.review_body AS review_body,
+                reviews.category AS category,
+                reviews.created_at AS created_at,
+                reviews.votes as votes,
+                COUNT(comments.review_id) AS comment_count
+            FROM reviews
+            INNER JOIN comments
```

Comment:
Is an INNER JOIN correct here? ( INNER JOIN will only return reviews that have comments. )
Since your SELECT is selecting from reviews, that is your LEFT table.
You will want all reviews - even those with no comments, so what is a better type of join to use?